slam3d/core/Types.hpp

```diff
@@ -80,6 +107,7 @@ namespace slam3d
 	public:
 		Measurement(const std::string& r, const std::string& s,
 		            const Transform& p, const boost::uuids::uuid id = boost::uuids::nil_uuid());
+		Measurement(){};  // <- needed for de-serilaization
```

Comment:
Where is this needed? Is the de-serialization method probably missing?

Reply:
it is needed to create an empty measurement in order to use it with the boost archive

Reply:
Where do you get the error when the default constructor is not present? All tests work for me without it.

Reply:
in the neo4j test, when I have to deserialize a PointCloudMeasurement binary string into a Measurement::Ptr instead of an PointCloudMeasurement::Ptr

Reply:
We need this anonymous de-serialization in the classes implementing containers for measurement pointers, as they should not have a dependency to the actual measurement implementations.